app/api/user/service.py

```diff
@@ -1,12 +1,22 @@
 from uuid import UUID
+
 from sqlalchemy.orm import Session
-from . import models
+
+from .models import User
+from .schemas import UserUpdate
 
 
 def get_user_by_id(db: Session, user_id: UUID):
-    return db.query(models.User).filter(models.User.id == user_id).first()
+    return db.query(User).filter(User.id == user_id).first()
 
 
 def delete_user_by_id(db: Session, user_id: UUID):
-    db.query(models.User).filter(models.User.id == user_id).delete()
+    db.query(User).filter(User.id == user_id).delete()
+    db.commit()
+
+
+def update_user(db: Session, user_id: UUID, payload: UserUpdate):
+    db.query(User).filter(User.id == user_id).update(payload.dict(), synchronize_session="fetch")
```

Comment:
```suggestion
    db.query(User) \
      .filter(User.id == user_id) \
      .update(payload.dict(exclude_none=True), synchronize_session="fetch")

```

如果不將 `payload.dice()` 的 `exclude_none` 傳 `True` 的話，請求沒指定值的屬性或被設成 `None`，造成沒有要改變的欄位被設成 `None`。所以要將  `payload.dice()` 的 `exclude_none` 傳 `True` 將值是 `None` 的屬性拿掉